bin/reth/src/args/rpc_server_args.rs

```diff
@@ -127,6 +136,18 @@ pub struct RpcServerArgs {
     /// Gas price oracle configuration.
     #[clap(flatten)]
     pub gas_price_oracle: GasPriceOracleArgs,
+
+    /// Max number of bytes for cached block data
+    #[arg(long, default_value_t = DEFAULT_BLOCK_CACHE_SIZE_BYTES)]
+    pub block_cache_size_bytes: usize,
```

Comment:
these should be MB for convenience 

see:

https://github.com/paradigmxyz/reth/blob/c481b8717b40a9060e37dd90d49b3e1dfcc87c9e/bin/reth/src/args/rpc_server_args.rs#L107-L109

and helper function that converts:

https://github.com/paradigmxyz/reth/blob/c481b8717b40a9060e37dd90d49b3e1dfcc87c9e/bin/reth/src/args/rpc_server_args.rs#L133-L136